react/src/Menu.js

```diff
@@ -28,6 +29,7 @@ export function Menu(props) {
             <DealPublishMenuItem />
             <DealTransfersMenuItem />
             <InspectMenuItem />
+            <PiecesMenuItem />
```

Comment:
At the moment the data in PieceDirectory.js is mock data, so not sure if we want to expose it in the UI yet